manage-mysql-user.py

```diff
@@ -122,20 +150,92 @@ def test_connection(username, password_parameter):
         return True
 
 
+def validate_event(event):
+    is_valid = True
+
+    if "mysql_user_username" not in event.keys():
+        logger.error(f"Invalid event: 'mysql_user_username' must be set")
+        is_valid = False
+
+    if ("mysql_user_password_parameter_name" in event.keys()) is (
```

Comment:
It took me a moment to understand the logic here, so maybe add a comment to explain

Reply:
I guess it's the case where someone writing the code thinks it's self-explanatory where it actually isn't - I thought the error message in that block would explain what's going on.